blocks/header/header.js

```diff
@@ -85,6 +89,107 @@ function toggleMenu(nav, navSections, forceExpanded = null) {
   }
 }
 
+async function fetchSearchData() {
+  if (!window.searchData) {
+    const resp = await fetch(`/${window.location.pathname.split('/')[1]}/query-index.json`);
+    const json = await resp.json();
+    window.searchData = json;
+  }
+  return window.searchData;
+}
+
+async function search(value) {
+  const searchData = await fetchSearchData();
+  const hits = [];
+  for (let i = 0; i < searchData.data.length; i += 1) {
+    const e = searchData.data[i];
+    const text = [e.title, e.description].join(' ').toLowerCase();
```

Comment:
instead of the overhead of creating an array and then joining it, you could use a template string: 
```
   const text = `${e.title} ${e.description}`.toLowerCase();
```

Reply:
Makes sense, I combined it with the below filter suggestion - much more readable, thanks!